### PR TITLE
Generate RTS price data

### PR DESCRIPTION
**Pull request by rodrigomha**

### CODE REVIEW

File: scripts/get_templates.jl [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    set_device_model!(template_ed, HydroDispatch, FixedOutput)
+    set_device_model!(template_ed, HydroEnergyReservoir, HydroDispatchRunOfRiver)
+    return
+end
+
+function set_ptdf_line_template!(template_uc)
+    # Unbounded Branch for now
+    set_device_model!(
+        template_uc,
+        DeviceModel(Line, StaticBranchUnbounded, duals=[NetworkFlowConstraint]),
```
Comment (jd-lara) [inline scripts/get_templates.jl]: If this is unbounded then the duals are always 0.0

File: scripts/main_test.jl [hunk 2]
```diff
@@ -0,0 +1,185 @@
+using Pkg
```
Comment (jd-lara) [inline scripts/main_test.jl]: should we call this script price generation?

**Review by jd-lara — CHANGES_REQUESTED** [review]
The file organization needs a little bit of a change to separate the functions used for price generation and the script.

**Comment by rodrigomha** [comment]
@jd-lara @sourabhdalvi 

Here are the main scripts. I need to figure out how to access the forecast to export it and that will complete this PR.

**Comment by rodrigomha** [comment]
@jd-lara This is ready for an additional review

**Review by jd-lara — APPROVED** [review]

**Pull request merged**